admin_guide/diagnostics_tool.adoc

```diff
@@ -367,9 +378,9 @@ ifdef::openshift-origin[]
 endif::[]
 image via the Docker CLI.
 
-To do so, specify your cluster's inventory file and the *_health.yml_* playbook
-when running the following `docker run` command as a non-root user that has
-privileges to run containers:
+Specify your cluster's inventory file and the *_health.yml_* playbook when
```

Comment:
Could we just say
As non-root user that has privileges to run containers, run the following command: